tests/tests/hrrr_control

```diff
@@ -72,29 +54,39 @@ export LIST_FILES="atmos_4xdaily.tile1.nc \
                    RESTART/sfc_data.tile6.nc"
 
 export_fv3
+export NPZ=127
+export NPZP=128
+export DT_ATMOS=720
+export SYEAR=2021
+export SMONTH=03
+export SDAY=22
+export SHOUR=06
+export RESTART_INTERVAL="12 -1"
+export OUTPUT_GRID='gaussian_grid'
+export NSTF_NAME='2,0,0,0,0'
+export WRITE_DOPOST=.true.
+export IAER=5111
+export OUTPUT_FH='0 21 24'
 
-export FHMAX=24
-export OUTPUT_FH="12 -1"
+export FV3_RUN=control_run.IN
+export CCPP_SUITE=FV3_HRRR
+export INPUT_NML=rap.nml.IN
+export FIELD_TABLE=field_table_thompson_aero_tke
+export NEW_DIAGTABLE=diag_table_rap
 
-export DT_ATMOS="600"
+export DT_ATMOS="300"
```

Comment:
we can remove the quotes around all the 300 values for dt_atmos

Reply:
Thanks, Denise. I will remove all duplicates of dt_atmos and the double quotes around the remaining ones.